### ISSUE TITLE
Record the number of files in an archive during the import process

### ISSUE THREAD

**Issue by bareheiny** [mcpierce/comixed#538]
As a user, I would like to know the number of files a comic archive contains - so I can easily identify files that were imported with less pages than are contained in the archive.

This doesn't need to be exposed in the front end (at this time), a simple int field in the comic table would suffice.

Ideally, the count would exclude xml files to reduce the frequency of false positives when comparing (for example) the number of hashed pages to the number of files in an archive.

Currently, the only way I can complete this type of check is by counting the hashed pages per comic, and then comparing the result to the page count in ComicRack.

**Comment by mcpierce** [comment]
I don't think this would be a terribly hard thing to add. We could add a small section to the comic overview tab that shows the file count and, when clicked, could open up and show the list of filenames. What do you think of such an enhancement?

**Comment by bareheiny** [comment]
Sounds good to me.

I'm not so worried about it being in the front-end for now, as I'm using SQL to extract data from CX to compare to CR...being able to skip the comparison, and just query the CX database would be more convenient.

Mostly, I see this sort of comparison being a one-off use - specifically identifiying where problems may be in an existing library.  Although being able to find issues with small page counts is also useful - as that can indicate a incomplete archive.

**Comment by mcpierce** [comment]
@bareheiny This issue is now being tracked [here](https://github.com/comixed/comixed/issues/12) in the central ComiXed project now.
